roles/openshift_manage_node/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+- name: Handle unscheduleable node
+  delegate_to: "{{ openshift_first_master }}"
```

Comment:
Is delegate_to even needed here? Since openshift_manage_node is being called from a play that sets hosts to oo_first_master, in essence, it appears we are delegating this task to the host specified itself (which I would suspect is the reason that we do not see the serialization bug).